plugins/uf2/uf2.quirk

```diff
@@ -0,0 +1,156 @@
+# match all devices with this udev subsystem
+[BLOCK]
+Plugin = uf2
+
+# Raspberry Pi RP2 [no CURRENT.UF2]
+[USB\VID_2E8A&PID_0003]
+Guid = UF2\FAMILY_E48BFF56
+Flags = updatable,will-disappear
+
+# Adafruit Trinket
+[USB\VID_1781&PID_0C9F]
+Flags = updatable
+
+# Adafruit Trinket M0
+[USB\VID_239A&PID_001E]
+Flags = updatable
+
+# Adafruit Feather M0 Express
+[USB\VID_239A&PID_001B]
+Flags = updatable
+
+# nRF52840 MDK
+[USB\VID_239A&PID_0029]
+Flags = updatable
+
+[UF2\FAMILY_16573617]
```

Comment:
where did all the family values come from?  Maybe need a registry import helper script that is part of the release process.

Reply:
From here: https://github.com/microsoft/uf2/blob/master/utils/uf2families.json -- it's not often updated, and it's only really NTH. I've also fixed-up some of the vendor/names as we have two properties rather than just a string. I'll add that as a comment, but I'm not sure it's worth the pain to make it auto-update.